substrate/frame/support/procedural/src/lib.rs

```diff
@@ -186,133 +186,12 @@ pub fn construct_runtime(input: TokenStream) -> TokenStream {
 	construct_runtime::construct_runtime(input)
 }
 
-/// The pallet struct placeholder `#[pallet::pallet]` is mandatory and allows you to specify
-/// pallet information.
+#[rustfmt::skip]
```

Comment:
Why do we need this?

Reply:
Otherwise it wraps the doc link over multiple lines, which breaks it